x-pack/plugins/apm/public/components/app/service_groups/service_groups_list/service_group_card.tsx

```diff
@@ -73,9 +76,27 @@ export function ServiceGroupsCard({
     href,
   };
 
-  return (
-    <EuiFlexItem key={serviceGroup.groupName}>
-      <EuiCard layout="vertical" {...cardProps} />
-    </EuiFlexItem>
+  const Card = <EuiCard layout="vertical" {...cardProps} />;
```

Comment:
Nit: personally i prefer if this was inlined.

Reply:
WDYT https://github.com/elastic/kibana/pull/128068/commits/5bdf4f9477fe080204b03bd50ebe809520abd285?